pkg/antctl/command_list.go

```diff
@@ -140,7 +145,14 @@ func (cl *commandList) GetDebugCommands(mode string) [][]string {
 		}
 		if mode == runtime.ModeController && cmd.supportController ||
 			mode == runtime.ModeAgent && cmd.supportAgent {
-			allCommands = append(allCommands, strings.Split(cmd.cobraCommand.Use, " ")[:1])
+			var currentCommand []string
+			if group, ok := groupCommands[cmd.commandGroup]; ok {
+				currentCommand = append(currentCommand, group.Use)
+				currentCommand = append(currentCommand, cmd.cobraCommand.Use)
+				allCommands = append(allCommands, currentCommand)
```

Comment:
`currentCommand` should be declared in this block if it's only used in it.
But to be more generic and reuse more code, it could be similar to L132-L137:
```
var currentCommand []string
if group, ok := groupCommands[cmd.commandGroup]; ok {
        currentCommand = append(currentCommand, group.Use)
}
currentCommand = append(currentCommand, strings.Split(cmd.cobraCommand.Use, " ")[:1])
allCommands = append(allCommands, currentCommand)
```

Reply:
`strings.Split(cmd.cobraCommand.Use, " ")[:1]` is a string slice and can't be append to another slice directly, not sure what's purpose to split `cmd.cobraCommand.Use` originally, I will check how to reuse code. thanks.

Reply:
looks like `strings.Split(cmd.cobraCommand.Use, " ")[:1]` is only converting `cmd.cobraCommand.Use` into a string slice so it can be appended to `allCommands` which is two dimensions array, so I change the code just the same as  L132-L137 which will do the same thing once there is `currentCommand` string slice defined.

Reply:
My mistake. It could use strings.Split(cmd.cobraCommand.Use, " ")[0]. It splitted it because the "use" of other commands includes some arguments.
I guess the current commit will cause `supportbundle` command fail.